mlir/include/mlir/Dialect/Vector/IR/VectorOps.td

```diff
@@ -2559,7 +2559,7 @@ def Vector_MaskOp : Vector_Op<"mask", [
                                  Location loc);
   }];
 
-  let hasCanonicalizer = 1;
+  let hasCanonicalizer = 0;
```

Comment:
Do we need this line anymore? I thibk we can remove it.

Reply:
Yes, I'm adding that back in my next PR but I removed it from now.